src/components/notesEditor/notesEditor.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Archiveicon, Coloricon, Tagicon, Trashicon } from '../assets/svg'
+import "./noteseditor.css"
+
+function Noteseditor() {
+  return (
+    <div className='note-editor-card'>
+      <div className='note-editor-title' contentEditable={true}>Title</div>
+      <div className='note-editor-body' contentEditable={true}>Take a Note...</div>
```

Comment:
here instead of div you can use an input tag and put "take a value..." as a placeholder and not directly as its value